### PR TITLE
docs: introduce `prettier` and `md-padding` to format all docs

### PR DESCRIPTION
**Pull request by spencercjh**
I found that the project lacks some automated mechanism to format the document.

### CODE REVIEW

**Comment by ccoVeille** [comment]
Please fix commit message and PR title. There is a typo, it's not pettier but prettier

**Comment by spencercjh** [comment]
> Please fix commit message and PR title. There is a typo, it's not pettier but prettier

OMG 😮 

DONE.

**Comment by spencercjh** [comment]
> Very useful tool. Thank you! I left some comments, looks like it breaks the [GitHub-flavored alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)

This is an issue that has not been resolved by the prettier community. The solution is as follows: https://github.com/prettier/prettier/issues/16454.

I don't know if you can accept it.

**Comment by hengyoush** [comment]
> > Very useful tool. Thank you! I left some comments, looks like it breaks the [GitHub-flavored alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)
> 
> This is an issue that has not been resolved by the prettier community. The solution is as follows: [prettier/prettier#16454](https://github.com/prettier/prettier/issues/16454).
> 
> I don't know if you can accept it.

you mean "add an empty line between note and text" ? 

```
> [!IMPORTANT]
>
> **Lorem ipsum dolor sit amet, consectetur adipiscing elit**
```

I can accept this.

File: docs/how-to-build.md [hunk 1]
```diff
@@ -13,53 +15,76 @@ This document describes the local compilation method for kyanos. My environment
...
 ```
 
-the kyanos executable file will be generated in the root directory of the project.
+the kyanos executable file will be generated in the root directory of the
+project.
 
+<!-- prettier-ignore -->
 > [!IMPORTANT]
-> Note that this binary file does not include the BTF files from [btfhub-archive](https://github.com/aquasecurity/btfhub-archive/). If you run this kyanos on a lower version kernel without BTF support, it may fail to start. You can build a kyanos artifact with embedded BTF files using the following commands:
-> ::: code-group
```
Comment (hengyoush) [inline docs/how-to-build.md:47]: should add `<!-- prettier-ignore -->` before `::: code-group`

**Comment by hengyoush** [comment]
LGTM

**Pull request merged**